### ISSUE TITLE
Question: is HEIC not working properly?

### ISSUE THREAD

**Issue by pdgonzalez872** [elixir-plug/mime#37]
Hi, sorry if this is not applicable somehow, but I thought I'd reach out to verify something:

```
iex(1)> MIME.type("png")
"image/png"
iex(2)> MIME.type("heic")
"application/octet-stream"
```

This seems to be supported though: https://github.com/elixir-plug/mime/blob/master/priv/mime.types#L1642-L1643

System: Elixir 1.10.3, Erlang 23. MIME version is `1.3.1`.

Is this expected behavior? Thank you in advance!

**Comment by josevalim** [comment]
There was no new release after those were added but I just shipped v1.4.0! :+1:

**Comment by pdgonzalez872** [comment]
perfect! Thanks!
